immichpy/cli/commands/workflows.py

```
@@ -20,45 +21,36 @@
-    filters: list[str] = typer.Option(
-        ...,
-        "--filters",
-        help="""Workflow filters
-
-As a JSON string with keys: filterConfig (object), pluginFilterId (string)""",
+    name: str | None = typer.Option(None, "--name", help="""Workflow name"""),
+    steps: list[str] | None = typer.Option(
+        None,
+        "--steps",
+        help="""As a JSON string with keys: config (object), enabled (boolean), method (string)""",
     ),
```

Ruff (0.15.20)

[warning] Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)